sale.py

```diff
@@ -103,16 +103,16 @@ def create_invoice(self, invoice_type):
 
         invoice = super(Sale, self).create_invoice(invoice_type)
 
-        if (invoice
-                and invoice_type == 'out_invoice'
-                and self.shipment_cost_method == 'shipment_capped'):
+        if (invoice and
```

Comment:
`if (
    invoice and invoice_type == 'out_invoice'
    and self.shipment_cost_method == 'shipment_capped'
):`